mvtools/terraform-iac/ec2-userdata.tftpl

```diff
@@ -45,9 +45,11 @@ mkdir -p /root/code
 aws s3 cp s3://${s3_bucket}/code.zip /tmp/code.zip
 unzip /tmp/code.zip -d /root/code/
 aws s3 cp s3://${s3_bucket}/results.csv /root/code/results.csv 
-python3 /root/code/mvtools/python-survey-app/questionnaireresults.py
+python3 root/code/mvtools/python-survey-app/questionnaireresults.py > /root/code/clean_results.csv
+python3 questionnaireresults.py 2> /root/code/error.txt
```

Comment:
This is good! But it needs the same path as the rest of the code. In your case root/code/mvtools/python-survey-app/questionnaireresults.py, not the file itself, as it won't find it.